Fix error in Pod Overhead documentation (#34337)
kubernetes/website: content/en/docs/concepts/scheduling-eviction/pod-overhead.md

requests and the overhead, then looks for a node that has 2.25 CPU and 320 MiB of memory available.

Once a Pod is scheduled to a node, the kubelet on that node creates a new {{< glossary_tooltip

> text="cgroup" term_id="cgroup" >}} for the Pod. It is within this cgroup that the underlying

Reviewer: ```suggestion
text="cgroup" term_id="cgroup" >}} for the Pod. It is within this cgroup that the underlying
```
The proposed change of this word from pod to cgroup is tricky since a container isn't necessarily created within a cgroup but with the use of cgroups and namespaces (uts, network, pid, user, ipc)

My previous lgtm is from a SIG Docs view
It would be great if have a technical review from @kubernetes/sig-node-pr-reviews or a sme like @mikebrow

Reviewer 2: I'm not sure the rewording here is super helpful. For most people's mental model, I think keeping this as `pod` probably makes more sense.

Reviewer 3: quick skim to this point of the Pod Overhead documentation...

The ```{{< glossary_tooltip
 text="cgroup" term_id="cgroup" >}}``` should also be included above on line 28 where it is initially referenced 

```suggestion
text="cgroup" term_id="cgroup" >}} with the resource limits calculated for the Pod. It is within the pod's cgroup limits that the underlying container runtime creates the pod and it's containers.
```

Reviewer: I agree with @endocrimes 's [comment ](https://github.com/kubernetes/website/pull/34337#discussion_r930902759) to keep the word `pod` instead in line 108

> text="cgroup" term_id="cgroup" >}} with the resource limits calculated for the Pod. The pod's cgroup limits are applied, by the underlying container runtime, to the pod and containers in the following manner.

Reviewer 4: nit: drop the commas
```suggestion
text="cgroup" term_id="cgroup" >}} with the resource limits calculated for the Pod. The pod's cgroup limits are applied by the underlying container runtime to the pod and containers in the following manner.
```

Reviewer 4: nit: this is a sentence fragment - `... in the following manner` should end in a colon `:`

> container runtime will create containers.

Reviewer 4: did you mean to delete this line?
